docs/usage.rst

```diff
@@ -199,31 +199,44 @@ or simply use ``fulfilled_creation_tx``:
 
     In [0]: creation_tx = fulfilled_creation_tx
 
-Preparing the transfer transaction:
+In order to prepare the transfer transaction, we first need to know the id of
+the asset we'll be transferring. Here, because Alice is consuming a ``CREATE``
+transaction, we have a special case in that the asset id is NOT found on the
+``asset`` itself, but is simply the ``CREATE`` transaction's id:
 
 .. ipython::
 
-    In [0]: cid = 0
+    In [0]: asset_id = creation_tx['id']
 
-    In [0]: condition = creation_tx['conditions'][cid]
+    In [0]: transfer_asset = {
+       ...:     'id': asset_id,
+       ...: }
+
+Let's now prepare the transfer transaction:
+
+.. ipython::
+
+    In [0]: fulfilled_output_id = 0
```

Comment:
How about calling it `index` instead of `id` to make it clearer, since it is really the `index` of the element of a list.